syft/pkg/cataloger/php/parse_composer_lock.go

```diff
@@ -40,6 +45,11 @@ func parseComposerLock(_ source.FileResolver, _ *generic.Environment, reader sou
 				),
 			)
 		}
+
+		// TODO: did we omit this on purpose?
```

Comment:
bump

Reply:
This was a comment when I was exploring all the catalogers for their different license implementations

- I can follow this TODO up with and issue like we have done with other things surfaced in this PR